### PR TITLE
Dev: cibconfig: remove related code about detecting crm_diff support --no-version

### PR DESCRIPTION
**Pull request by liangxin1300**
According issue #759 , and consider from very early version of pacemaker, crm_diff has `--no-version`, remove unneeded related codes in this PR

### CODE REVIEW

File: crmsh/cibconfig.py [hunk 1]
```diff
@@ -2669,20 +2658,8 @@ def _patch_cib(self, force):
...
-                self._crm_diff_cmd = 'crm_diff --no-version'
-            else:
-                self._crm_diff_cmd = 'crm_diff'
-
         self._copy_cib_attributes(current_cib, self.cib_orig)
         current_cib = None  # don't need that anymore
-        # only bump epoch if we don't have support for --no-version
-        if not self._crm_diff_cmd.endswith('--no-version'):
-            # now increase the epoch by 1
-            self.bump_epoch()
```
Comment (liangxin1300) [inline crmsh/cibconfig.py:2685]: `bump_epoch` only used once, so deleted it, too

**Comment by liangxin1300** [comment]
@gao-yan Could you please review this PR? Thanks!
I find that since pacemaker 1.1.16, crm_diff support --no-version, so I plan to backport this PR to crmsh-3.0 and crmsh-4.1

**Review by gao-yan — APPROVED** [review]
Looks good.

**Comment by gao-yan** [comment]
> @gao-yan Could you please review this PR? Thanks!
> I find that since pacemaker 1.1.16, crm_diff support --no-version, so I plan to backport this PR to crmsh-3.0 and crmsh-4.1

Makes sense.

**Pull request merged**